buildtools/commands/test.js

```diff
@@ -0,0 +1,73 @@
+const { resolve } = require('path');
+const { execSync } = require('child_process');
+
+exports.command = 'test';
+exports.desc = 'Runs tests for the Okta Sign-in Widget';
+exports.builder = {
+  type: {
+    description: 'Type of the test runner',
+    choices: [
+      'karma',
+      'jest',
+    ],
+    required: true,
+  },
+  config: {
+    description: 'Optional override for the test config file',
+  },
+  suiteHelp: {
+    description: 'Run this to see the test CLI options',
+  },
+};
+
+const suiteMap = {
+  karma: {
+    bin: 'karma start',
```

Comment:
nit:  Should we call this `startCmd`, `testStartCmd` or `testCmd` ?

Reply:
Great idea. Will change to `startCmd` to be more clear!